lib/bootstrap.ts

```diff
@@ -0,0 +1,12 @@
+import dotenv from 'dotenv';
+
+export default () => {
+  /** Loads environment variables from the file .env */
+  dotenv.config();
+
+  /** rethrow unhandled promise rejection, to crash process of an unwrapped instance */
```

Comment:
Shouldn't the TypeDoc comments start with uppercase letters? Do we have some kind of code style guidelines for this?

Reply:
Fixed.

I don't think we have guidelines for this.

Reply:
I think we should just be following TypeDoc conventions. We do have some basic contribution guidelines, I can link to some TypeDoc guides from there.